README.md

```diff
@@ -197,9 +213,40 @@ output instead:
 ,,,1554210895
 ```
 
-It is possible to control the number of records to be put in a
-particular output file by setting `file.max.records`. By default, it is
-`0`, which is interpreted as "unlimited".
+**NB!**
+
+ - The `key.converter` property must be set to `org.apache.kafka.connect.converters.ByteArrayConverter`
+or `org.apache.kafka.connect.storage.StringConverter` for this data format.
+
+ - The `value.converter` property must be set to `org.apache.kafka.connect.converters.ByteArrayConverter` for this data format.
+
+#### **JSONL** Format example
+
+For example, if we output `key,value,offset,timestamp`, a record line might look like:
+
+```
+{ "key": "k1", "value": "v0", "offset": 1232155, "timestamp":"2020-01-01T00:00:01Z" }
+```
+
+OR
+
+```
+{ "key": "user1", "value": {"name": "John", "address": {"city": "London"}}, "offset": 1232155, "timestamp":"2020-01-01T00:00:01Z" }
+```
+
+Unfortunately, the order of the fields for this data format cannot be guaranteed.
```

Comment:
I'd remove this note, as it's not normally expected from JSON.

Reply:
Ok
- [ ] Remove field order guarantee

Reply:
- [ ] Remove